components/Cards/SchoolCard/SchoolCard.css

```diff
@@ -1,23 +1,139 @@
 .SchoolCard {
-  max-width: 400px;
-  height: 550px;
+  width: 330px;
   font-family: var(--primaryFontFamily), sans-serif;
+  background-color: #ffffff;
```

Comment:
Can you use the white variable?

Reply:
If you're talking about `var(--white)`, it's not `#fff`. The logos have pure white backgrounds. Still want me to make the change?

![logo-white-bg](https://user-images.githubusercontent.com/2691129/56766377-46511980-6777-11e9-8e7c-0d11e6286d7e.png)

Reply:
ugh gross - no, good shout didn't realize.

Can you write a comment in the CSS concerning that?